blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/BluefloodServiceStarter.java

```diff
@@ -295,8 +303,10 @@ public void run() {
                     TimeUnit.MINUTES.toMillis(savePeriodMins));
         }
 
-        // has the side-effect of causing static initialization of Metrics, starting instrumentation reporting.
-        new RestartGauge(Metrics.getRegistry(), RollupService.class);
+        if (shouldInstantiateRestartGauge) {
```

Comment:
I agree with @ChandraAddala. If all you want to do is skip the call of RestartGauge, then mock it.